src/adaptation/index.ts

```diff
@@ -0,0 +1,69 @@
+import { NativeModules } from 'react-native';
+import BatchedBridge from 'react-native/Libraries/BatchedBridge/BatchedBridge';
+import NativeInstance from '../nativeInstance';
+import { VideoAdaptationData, AdaptationConfig } from './adaptationConfig';
+
+// Export config types from Adaptation module.
+export { VideoAdaptationData, AdaptationConfig };
```

Comment:
Hi @saravanans-github!
To fix the documentation error, and possibly a potential integration error regarding imports, please replace this line here with the suggestion.

```suggestion
export * from './adaptationConfig';
```

The reason is that all exported types should be re-exported on the top-level and now this is not the case.